.github/release-examples.md

```
@@ -1,4 +1,4 @@
-## Sapphire 1.0.8: New Syntax & Examples
+## Sapphire: Syntax & Examples
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)